<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareToolTreeView</class>
 <widget class="QTreeView" name="CompareToolTreeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompareToolTreeView</string>
  </property>
  <action name="action_Move">
   <property name="text">
    <string>移动到右边</string>
   </property>
  </action>
  <action name="action_Del">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Output">
   <property name="text">
    <string>导出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
